client/src/test/java/io/moquette/spi/impl/security/AESTest.java

```diff
@@ -0,0 +1,41 @@
+package io.moquette.spi.impl.security;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+public class AESTest {
+
+    @Test
+    public void testEncrypted() {
+        byte[] aesKey = new byte[]{-31, 117, 20, -91, 78,
```

Comment:
Rename this variable `encrypted`

Reply:
I thought you meant the method name :)
Done